packages/api/amplify_api_android/CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
-## 0.4.1 (2022-03-04)
```

Comment:
why overwrite the old entries here? same for ios?

Reply:
Hum, I think all the new packages of `api` category have never been released. This original changelog might've been added wrongly, it seems resonable to override here.